### DISCUSSION TITLE
How can I get rid of these backslashes?

### DISCUSSION THREAD

**Discussion by alpinemobile** [elfmz/far2l#2071] [General]
Hi. First of all thanks for the great work with FAR2L .
I am renaming a file and I press F6 , then I press SHIFT ENTER to fill-in the name and I get a backslash character in front of each whitespace.
![image](https://github.com/elfmz/far2l/assets/13587452/5dd32c89-69a4-4fad-b775-afb6fa26c0a8)

How can I get rid of the backslashes, as they are taken into account if I don't remove them?
Thanks!

**Comment by akruphi** [comment]
Thanks for mentioning this problem. PR https://github.com/elfmz/far2l/pull/2081 was merged. You can rebuild far2l and check now.
